packages/admin/src/Charcoal/Admin/ServiceProvider/AdminServiceProvider.php

```diff
@@ -181,6 +186,34 @@ protected function registerAdminServices(Container $container)
                 return $adminUrl;
             };
         }
+
+        /**
+         * Overwrite view instance.
+         *
+         * @param GenericView $view The view instance.
+         * @param Container $container A container instance.
+         * @return ViewInterface
+         */
+        $container->extend('view', function (GenericView $view, Container $container): GenericView {
+            return new GenericView([
+                'engine' => $container['view/engine/mustache']
+            ]);
+        });
```

Comment:
This should be type-hinted to the view interface to allow for custom implementations.